pkg/operator/ceph/file/controller.go

```diff
@@ -224,6 +225,10 @@ func (c *FilesystemController) filesystemOwner(fs *cephv1.CephFilesystem) metav1
 }
 
 func filesystemChanged(oldFS, newFS cephv1.FilesystemSpec) bool {
+	if cmp.Equal(oldFS.MetadataPool.Replicated.Size, newFS.MetadataPool.Replicated.Size) {
```

Comment:
@travisn will this be a good practice to use `cmp.Equals`?.
Also, if I try to change the replicas in `filesystem.yaml` after making above changes in the go code (hoping changes will be reflected), the changes are not reflected at all.
Can you please drop some more hints maybe?.
Thanks

Reply:
This basic comparison is preferring for ints:
```
oldFS.MetadataPool.Replicated.Size == newFS.MetadataPool.Replicated.Size
```

Reply:
In the method `doFilesystemCreate()` looks like the filesystem will not be updated if it already exists. So we'll need to change this to update the pools if the size changed.